source/client.c

```diff
@@ -77,6 +77,38 @@ void aws_mqtt_client_clean_up(struct aws_mqtt_client *client) {
     AWS_ZERO_STRUCT(*client);
 }
 
+/* Reconnect helpers */
+static void s_attempt_reconnect(struct aws_task *task, void *userdata, enum aws_task_status status);
+
+int s_create_reconnect_task(struct aws_mqtt_client_connection *connection) {
+    AWS_ASSERT(!connection->reconnect_task);
```

Comment:
you can add a string here if you'd like:
AWS_ASSERT(!connection->reconnect_task && "my string I want the console output to show")